src/app/checkout/step-2/bank-account/bank-account.tpl.html

```diff
@@ -82,7 +82,7 @@ <h4 class="panel-title  border-bottom-small  visible" translate>Bank Account Pay
             <div class="panel-body">
               <div class="row">
                 <div class="col-sm-8">
-                  <img src="/assets/img/check-sample.png" alt="Credit Card Icon"/>
+                  <img src="/assets/images/check-sample.png" alt="Credit Card Icon"/>
```

Comment:
Because this will be hosted on AEM, these are going to have to be absolute paths.  I can set a window variable for the domain, then they can be referenced via ng-src.  Thoughts?